### PR TITLE
Mots 143 view locations panel

### PR DESCRIPTION
**Pull request by kkaczmarczyk**

### CODE REVIEW

File: src/webapp/js/container/chiefdoms-table.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  {
+    Header: 'Name',
+    accessor: 'name',
+  }, {
+    Header: 'Parent District',
+    accessor: 'parent',
+  },
+];
+
+class ChiefdomsTable extends Component {
```
Comment (pmuchowski) [inline src/webapp/js/container/chiefdoms-table.js]: I think you can add some base table for all of the locations and pass columns and location list as a parameter to remove this duplicated code

File: src/main/java/org/motechproject/mots/web/LocationController.java [hunk 2]
```diff
@@ -33,4 +38,56 @@
...
   }
+
+  /**
+   * Get list of districts for preview.
+   * @return list of all districts
+   */
+  @RequestMapping(value = "/districtsOnly", method = RequestMethod.GET)
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public Set<LocationPreviewDto> getDistrictsOnly() {
```
Comment (pmuchowski) [inline src/main/java/org/motechproject/mots/web/LocationController.java]: did you think about the way to reuse the already fetched list (this used in chw and incharges)?

Comment (kkaczmarczyk) [reply src/main/java/org/motechproject/mots/web/LocationController.java]: there are 4 separate tabs for each type of location, and the list used in chw and incharges is fully hierarchic

File: src/webapp/js/container/chiefdoms-table.js (hunk 1 above)
Comment (kkaczmarczyk) [reply src/webapp/js/container/chiefdoms-table.js]: probably - will look into that

File: src/webapp/js/container/locations-table.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  };
+}
+
+export default connect(mapStateToProps, { fetchLocationsOfType })(LocationsTable);
+
+LocationsTable.propTypes = {
+  fetchLocationsOfType: PropTypes.func.isRequired,
+  locationsList: PropTypes.arrayOf(PropTypes.shape({
+  })).isRequired,
+  tableColumns: PropTypes.func.isRequired,
```
Comment (pmuchowski) [inline src/webapp/js/container/locations-table.js]: use arrayOf(PropTypes.shape({})) here not "func"

File: src/webapp/js/container/locations-table.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  locationType: PropTypes.func.isRequired,
```
Comment (pmuchowski) [inline src/webapp/js/container/locations-table.js]: use "string" instead of "func"

**Pull request merged**